go-controller/pkg/cni/cniserver.go

```diff
@@ -137,10 +138,36 @@ func cniRequestToPodRequest(cr *Request) (*PodRequest, error) {
 	if err != nil {
 		return nil, fmt.Errorf("broken stdin args")
 	}
+
 	req.CNIConf = conf
+	req.timestamp = time.Now()
 	return req, nil
 }
 
+func (s *Server) startSandboxRequest(req *PodRequest) error {
+	// Only sandbox add requests are tracked because only adds need
+	// to be canceled when a newer request comes in.
```

Comment:
or when a delete comes in. CNI Del doesn't need to be cancelled and may proceed as usual.